### DISCUSSION TITLE
https://www.yunyitang.me/friends/

### DISCUSSION THREAD

**Discussion by giscus** [yunyit/yunyit.github.io#25] [Comments]
# [友情链接](https://www.yunyitang.me/friends/index.html)

在这个独立博客式微的时代，友情链接的意义是什么？ 和 Medium、简书之类的写作平台不同，每一个独立博客都是一个信息孤岛。我们没有类似「你可能

https://www.yunyitang.me/friends/

**Comment by at-wr** [comment]
**✋ Hi!** 请问可以与您交换友链吗，已加上贵站链接

```
figure src="https://wrye.dev/assets/avatar.webp"
[Blog of Wr](https://wrye.dev/)
`Web dev, Linux, and other tech or casual stuff.`
```

**Reply by yunyit** [reply]
已添加 😊

**Comment by L33Z22L11** [comment]
申请添加友链。
```
- [纸鹿摸鱼处](https://blog.zhilu.cyou)
  - 纸鹿至麓不知路，支炉制露不止漉
```

**Reply by yunyit** [reply]
已添加 😊 新年快乐！

**Comment by sangmingming** [comment]
哈喽，申请交换友链，已经添加友链。
```    标题: 码农明明桑
    Url: https://isming.me
    头像: https://isming.me/chrome-512.png
    简介： 一个码农的个人的技术和日常记录
```

**Reply by yunyit** [reply]
已添加 😊 迟到的新年快乐！

**Comment by vensing** [comment]
我 [blog 站点](https://blog.youmuwhisper.space/)都快长草两年了，之前还换了个域名，已经破罐子破摔了。

博主竟然知道我的 blog 站点，还加了我的友链，真的好神奇好意外诶🤣

**Reply by yunyit** [reply]
哈哈应该是从别人的友链列表里找到的😄 握爪握爪~~~

**Comment by qiukuip** [comment]
博主你好，申请友链，已添加贵站链接。
- [lozhu's blog](https://lozhu.happy365.day)
  - 一切皆是因为好玩～

**Reply by yunyit** [reply]
抱歉前阵子忙，已添加～～

**Comment by yefoenix** [comment]
- [樽前小筑](https://yefoenix.ws/)
  - 酒类写作及博主的一些折腾

申请友链，
贵站链接已添加，请见 https://yefoenix.ws/blogroll

**Reply by yunyit** [reply]
抱歉前阵子忙，已添加～～

**Comment by C-Jeril** [comment]
加油
有机会一起交流
https://kuakua.app/

**Comment by elkan1788** [comment]
博主是不是把我的站点下线啦，请检查一下哦，谢谢。

鄙站友链：https://lisenhui.cn/flinks.html

**Reply by yunyit** [reply]
抱歉前阵子忙，已更新～～

**Comment by seminorm** [comment]
您好，很感谢添加了本站(https://hugo.bnblogs.cc)的友链，由于服务器到期，本站即将不可访问，上线时间不定，如果介意请注意移除本站友链，后续上线后我会再次申请友链！

**Comment by ZhouyiStudio** [comment]
- [Zhouyi's Blog](https://zhouyiblog.top)
  - 这个博客木有描述……

**Reply by ZhouyiStudio** [reply]
✋ Hi! 请问可以与您交换友链吗，已加上贵站链接

**Reply by yunyit** [reply]
刚访问的时候显示`503 Service Unavailable`，等会儿我再试下～

**Comment by AmosHuKe** [comment]
@yunyit 
Hi~ 以前的域名已经换掉咯，麻烦有空的时候更替一下~
```diff
- [Amos’s Blog](https://amoshk.top/)
+ [Amos](https://amooos.com/)
```

**Reply by yunyit** [reply]
好的～

**Comment by AllenWu233** [comment]
博主你好！申请交换友链，已添加贵站链接
```
- [逍遥乡 | Allen's World](https://blog.allenwu233.com/zh)
  - 天地与我并生，而万物与我为一。
```

**Reply by yunyit** [reply]
已添加 😊

**Comment by liangbm3** [comment]
申请友链
- [liangbm3's blog](https://liangbm3.top/)
  - 总有人间一两风，填我十万八千梦！

**Reply by yunyit** [reply]
已添加 😊

**Comment by Reiziuh** [comment]
Hi，交换个友链吧。顺祝马年吉祥，gung1 hei2 faat3 coi4，lei6 si6 dau6 loi4 :)

名称 橙树志 | citydatum
网址 https://citydatum.cn
图标 https://citydatum.cn/favicon.ico
描述 过已过，来未来，唯当下

**Reply by yunyit** [reply]
迟到的新年快乐！已添加 😊

**Comment by SuperDemon921** [comment]
您好，我是一名专攻网络安全方向的本科生，在网上看到您的网站，觉得很高质量，已将您的网站加入本站友链，现在申请将本站链接加入您的友链中，望同意：
名称：JerryGao's Blog
地址：https://www.jerrygao.cn
图片：https://www.jerrygao.cn/icons/icon-512.png
简介：爱折腾的安全小白​

**Reply by yunyit** [reply]
已添加 😊

**Comment by TooManyDe** [comment]
交换友链
名称：的的不休
网址：https://ddbx.org
图标：https://cdn.ddbx.org/12.png
描述：每一段旅行都有终点

**Comment by AllenWu233** [comment]
博主您好！我迁移到了新域名，麻烦您更新一下吧：

```diff
- - [逍遥乡 | Allen's World](https://blog.allenwu233.com/zh)
+ - [Allen@冰轮韵](https://lunarithm.space)
-  - 天地与我并生，而万物与我为一。
+  - 冰轮舞韵，代码凝魂。
```

**Comment by MarkXuJQ** [comment]
您好啊！很喜欢你的文字，可以交换一个友链嘛？
{
  "name": "Mark 的自留地",
  "url": "https://markxu.icu",
  "avatar": "https://markxu.icu/images/IMG_1766.JPG",
  "description": "慢慢更新，慢慢打磨，认真分享，认真生活"
}
